tendrl/commons/__init__.py

```diff
@@ -85,7 +85,7 @@ def _validate_ns_obj_definitions(self, raw_ns, defs):
         if self.current_ns.objects:
             LOG.info("Validating registered (.py) objects in %s.objects", raw_ns)
             defined_objs = defs.get("objects", {})
-            regd_objs = [obj_name for obj_name in self.current_ns.objects if not self._get_object(obj_name).internal]
+            regd_objs = [obj_name for obj_name in self.current_ns.objects if not hasattr(self._get_object(obj_name), "internal") and not obj_name.startswith('_')]
```

Comment:
Etcd classes are not registered by tendrl in Namespace, please remove the "_" check